.travis.yml

```diff
@@ -24,6 +29,9 @@ script: |
     mvn clean verify --settings maven-settings.xml
   fi
 
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
The script at the url is pretty large and may change over time.  I think this request is a bit excessive because that would also mean that we should examine and put every single maven dependency in our repo because that code could also echo environment variables or send it to a remote server.

Reply:
True enough but Maven artifacts are signed. This is taking arbitrary code from a URL with no verification, and anyone can stick a proxy in between you and the server. It's always debatable how paranoid one needs to be, but the recent trend in piping the contents of URLs into bash is really a security hole.

Reply:
Note that it's also unlikely we'd ever know if someone got the keys, which then compromises the signing system of Maven. The security problem is contagious.

Reply:
Ah... but it's an https URL so there's a certificate for a particular domain involved. Maybe it's not so insecure then...